example/provider-extensions/garden/project/credentials/infrastructure-secrets.yaml.tmpl

```diff
@@ -0,0 +1,8 @@
+##########
+# Please use this template to maintain your infrastructure secrets and
+# save the resulting file at infrastructure-secrets.yaml file in the same directory.
+# You can find a documentation in the respective gardener extension.
+# AWS: https://github.com/gardener/gardener-extension-provider-aws/blob/master/docs/usage-as-end-user.md
+# Azure: https://github.com/gardener/gardener-extension-provider-azure/blob/master/docs/usage-as-end-user.md
+# GCP: https://github.com/gardener/gardener-extension-provider-gcp/blob/master/docs/usage-as-end-user.md
```

Comment:
Why do you list only those three? Either list all of none.

Reply:
I removed them all, otherwise there would be a ton of links.